From 9dc9e26b9196111eeacb5609b36957f74f5a14c4 Mon Sep 17 00:00:00 2001
From: Ismael Asensio <isma.af@gmail.com>
Date: Tue, 22 Dec 2020 22:23:17 +0100
Subject: [PATCH] RenameDialog: Add missing nullptr initialization

It fixes a crash when pressing `Apply to All` in those cases where
the action was not present, as the code checks for nullptr

BUG: 430374
FIXED-IN: 5.78
---
 src/widgets/renamedialog.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/widgets/renamedialog.cpp b/src/widgets/renamedialog.cpp
index ac1d91d2..c3e04753 100644
--- src/widgets/renamedialog.cpp
+++ src/widgets/renamedialog.cpp
@@ -161,6 +161,7 @@ public:
         bCancel = nullptr;
         bRename = bSkip = nullptr;
         bOverwrite = nullptr;
+        bOverwriteWhenOlder = nullptr;
         bResume = bSuggestNewName = nullptr;
         bApplyAll = nullptr;
         m_pLineEdit = nullptr;
-- 
GitLab

